multipledispatch/tests/test_core.py

```diff
@@ -203,3 +203,16 @@ def f(x, y):
     assert foo.f(A(), A()) == 1
     assert foo.f(A(), C()) == 2
     assert foo.f(C(), C()) == 2
+
+
+# Hacky method to get benchmnarks namespaced
+class TestBenchMark(object):
```

Comment:
What is this for?

Reply:
Basically a holder to stick generated tests into, with decent names so that pytest can discover them appropriately.

sadly pytest-benchmark doesn't just provide a decorator.